### ISSUE TITLE
xpath in edge broken

### ISSUE THREAD

**Issue by sgreener** [webdriverio/webdriverio#3034]
**Environment (please complete the following information):**
 - **WebdriverIO version:** 4.13.1
 - **Mode:** WDIO Testrunner
 - **If WDIO Testrunner, running sync/async:** [e.g. sync/async]
 - **Node.js version:** 8.11.1
 - **NPM version:** 6.4.1
 - **Browser name and version:** Edge 18.something (october win 10 update version)
 - **Platform name and version:** Windows 10

**Describe the bug**
A valid XPath when used in edge for elements that don't exist report as 'Invalid Selector', this in turn prevents .isExisting() from reporting true/false as expected.

This works as expected in Chrome, Firefox and IE11

**To Reproduce**

```
describe('xpath in edge', function () {

    const XPathExistingElement = '//input[@id="lst-ib"]'; // main input field
    const XPathNoneExistantElement = '//div[@id="doesNotExist"]'; // will not exist!

    it('should work', function () {

        browser.url('https://www.google.com/');

        browser.element(XPathExistingElement).waitForVisible();
        const ValidXPath = browser.element(XPathExistingElement).isExisting();

        // broken in edge only (for element that does not exist) 
        const InvalidXPath = browser.element(XPathNoneExistantElement).isExisting(); // should return false

        console.log('valid:', ValidXPath);
        console.log('invalid:', InvalidXPath);

    });

});
```

**Expected behavior**

The .isExisting() should return true / false (in this case false) for the none existant element specified in the xpath.

the browser.element() is reporting a ;'The specified selector is invalid.' message

**Additional context**

The following bug thats not going to be fixed by microsoft edge team looks like it could be the root cause...

[Edge webdriver bug 19224420](https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/19224420/)

In a nutshell, edge webdriver is reporting  “invalid selector” (400) instead of “no such element” (404)

**Comment by christian-bromann** [comment]
This selector strategy is defined in the WebDriver spec and should be supported. This can't be fixed by WebdriverIO
